shuup/core/api/category.py

```diff
@@ -27,14 +28,36 @@ class CategorySerializer(TranslatableModelSerializer):
     visibility = EnumField(CategoryVisibility)
     image = serializers.SerializerMethodField()
 
-    class Meta:
-        model = Category
-        exclude = ("shops",)
+    def __init__(self, *args, **kwargs):
+        super(CategorySerializer, self).__init__(*args, **kwargs)
+        request = self.context.get("request")
+        if request.method in ["POST"]:
+            self.fields["image"] = Base64FileField(required=False, write_only=True)
+            self.fields["image_path"] = serializers.CharField(required=False, write_only=True)
+        else:
```

Comment:
`elif request.method == "GET"`, this prevents returning when method is `HEAD` or any other crazy string